README.md

```diff
@@ -45,37 +61,62 @@ You'll also need to install a couple things that involve Gits. To do this, you n
 3. Run `haxelib git polymod https://github.com/larsiusprime/polymod.git` to install Polymod.
 4. Run `haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc` to install Discord RPC.
+5. Optional: - Run `haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons` to update Flixel-Addons. This fixes the transition bug for zoomed out stage cameras.
 
 You should have everything ready for compiling the game! Follow the guide below to continue!
 
-At the moment, you can optionally fix the transition bug in songs with zoomed out cameras.
-- Run `haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons` in the terminal/command-prompt.
-
-### Ignored files
+# Adding `APIStuff.hx` into `/source`
 
-I gitignore the API keys for the game, so that no one can nab them and post fake highscores on the leaderboards. But because of that the game
-doesn't compile without it.
+![APIStuff.hx_picture](https://user-images.githubusercontent.com/72235309/112080256-00b24c80-8b58-11eb-9499-524bf45a00eb.png)
 
-Just make a file in `/source` and call it `APIStuff.hx`, and copy paste this into it
+The creator gitignored the API keys of the game so no one could post fake high scores onto the leaderboards in Newgrounds. Unfortunatly, because this game requires the `API` and `EncKey` values to compile, you will need to add a file called `APIStuff.hx` into `/source`. How do I do that?
 
+1. Create a new text file called `APIStuff.hx` inside of the `/source` folder.
+2. Copy the following text:
 ```haxe
 package;
-
 class APIStuff
 {
 	public static var API:String = "";
 	public static var EncKey:String = "";
 }
-
 ```
+3. Paste the text into the APIStuff.hx file and save the file.
 
-and you should be good to go there.
+You should be good from there! Now, onto compiling!
 
-### Compiling game
+# Compiling the Game
 
-Once you have all those installed, it's pretty easy to compile the game. You just need to run 'lime test html5 -debug' in the root of the project to build and run the HTML5 version. (command prompt navigation guide can be found here: [https://ninjamuffin99.newgrounds.com/news/post/1090480](https://ninjamuffin99.newgrounds.com/news/post/1090480))
+## HTML Building:
 
-To run it from your desktop (Windows, Mac, Linux) it can be a bit more involved. For Linux, you only need to open a terminal in the project directory and run 'lime test linux -debug' and then run the executible file in export/release/linux/bin. For Windows, you need to install Visual Studio Community 2019. While installing VSC, don't click on any of the options to install workloads. Instead, go to the individual components tab and choose the following:
+Compiling the game is rather simple for HTML5 builds.
+1. Open your machine's command prompt/terminal and navigate to your root folder of the game. [An easy guide can be found here!](https://ninjamuffin99.newgrounds.com/news/post/1090480)
+2. Type `lime build html5` to build the HTML5 version of the game.

```

Comment:
It would be nice to hint at the `-debug` flag so people can use it to debug the program.